Makefile

```diff
@@ -223,14 +225,18 @@ clone-license:
 prepublish-build: clean-lib clean-runtime-helpers
 	NODE_ENV=production BABEL_ENV=production $(MAKE) build-bundle
 	$(MAKE) prepublish-build-standalone clone-license
-	# We don't want to publish .d.ts files yet
-	rm -rf packages/*/dts
 
 prepublish:
 	$(MAKE) check-yarn-bug-1882
 	$(MAKE) bootstrap-only
 	$(MAKE) prepublish-build
 	IS_PUBLISH=true $(MAKE) test
+	# We don't want to publish TS-related files yet, except for @babel/types
+	rm -f packages/*/lib/**/*.d.ts{,.map}
```

Comment:
It's regenerated in the line right after :slightly_smiling_face: 
It was easier to delete and re-generate it, rather than writing a Makefile loop to delete only everything else.

You can see the `@babel/types` and `@babel/template` (which already uses ts) published using this PR: https://drive.google.com/file/d/1TSyN9huOsGBIekpRv9Yf50DQEPuL4dED/view?usp=sharing (extracted from our e2e tests on CircleCI)

Reply:
> It's regenerated in the line right after 🙂
nice, missed that